content/tagging/_index.md

```diff
@@ -38,49 +38,44 @@ A few best practices on tags:
     * Slashes
 
-    Other special characters get converted to underscores.
-    **Note**: A tag cannot end with a colon (e.g., `tag:`)
 2. Tags can be **up to 200 characters** long and support unicode.
 3. Tags are converted to lowercase.
-4. A tag can have a `value` or a `key:value` syntax:
-    **For optimal functionality, we recommend constructing tags that use the `key:value` syntax.** The key is always what precedes the first colon of the global tag definition, e.g.:
+4. A tag can be in the format `value` or `key:value`. For optimal functionality, **we recommend constructing tags in the `key:value` format.** Commonly used tag keys are `env`, `instance`, `name`, and `service`. The key is always what precedes the first colon of the global tag definition, for example:
+    
+    | Tag                      | Key                | Value            |
+    |--------------------------|--------------------|------------------|
+    | `service:database:mysql` | `service`          | `database:mysql` |
+    | `service_database:mysql` | `service_database` | `mysql`          |
 
-    * `role:database:mysql` is parsed as **key**:`role` , **value**:`database:mysql`
-    * `role_database:mysql` is parsed as **key**:`role_database` , **value**:`mysql`
-
-    Examples of commonly used metric tag keys are `env`, `instance`, `name`, and `role`.
-
-5. `device`, `host`, and `source` are **reserved tag keys** and cannot be specified in the standard way.
+5.  **Reserved tag keys** `device`, `host`, and `source` cannot be used in the standard way.
 
 6. Tags shouldn't originate from unbounded sources, such as EPOCH timestamps or user IDs. These tags may impact platform performance and billing.
 
-## Applying Tags
-
-Tags may be added using any (or all) of the following methods:
-
-* Agent Tags (`datadog.yaml`)
-* [DogStatsD][2] Tags
-* Integration/Check Tags (each check on the local host supports tags by editing the yaml)
-* Tags generated by other services such as [AWS][3], [Azure][4], [GCE][5], etc.
-* Tags in the [API][6] - note other endpoints support tags as well such as Events and Metrics
-* [Chef Roles][7] and [Puppet][8] Tags (Chef and Puppet use the API - this may obviously be extended to other configuration management tools by you or Datadog)
-* Manually adding tags using the [Infrastructure List][9] (hover over host->select "Inspect"->"Edit Tags")
-
-## Examples
-
-Here is an example of tags using the time-series chart editor. For the first screenshot, no tags have been applied, and we're observing average CPU across all hosts:
-
-{{< img src="tagging/Tags_1.png" alt="Tags_1" responsive="true" style="width:75%;">}}
+## Assigning Tags
 
-In this next example, we've applied a tag (`region:eastus`) that enables Datadog to look at CPU across the US East Region. We've used region as an example, but you could use any arbitrary tag, including application, service, environment, etc.
+Tags may be [assigned][21] using any (or all) of the following methods:
```

Comment:
I think we should build an array here that give a quick desctiption/detail/overview on how to assign tags with the following methods, i.e: 

| Method | Description |
| [Configuration Files] | Setup tags manually in your main agent configuration files, or in your integrations configuration file. |